src/diem/testing/miniwallet/config.py

```diff
@@ -66,8 +80,10 @@ def serve(self, client: jsonrpc.Client) -> threading.Thread:
         api: falcon.API = falcon_api(App(self.account, client, self.name, self.logger), self.enable_debug_api)
 
         def serve() -> None:
-            self.logger.info("serving at http://%s:%s" % (self.host, self.port))
-            waitress.serve(api, host=self.host, port=self.port, clear_untrusted_proxy_headers=True, _quiet=True)
+            self.logger.info(f"serving at {self.server_url}")
+            waitress.serve(
+                api, host="0.0.0.0", port=self.server_conf.port, clear_untrusted_proxy_headers=True, _quiet=True
```

Comment:
self.server_conf.host?

Reply:
it is confusing to have host config and then "0.0.0.0" used for serve.
I think we should keep existing host config.

Reply:
specifically used `0.0.0.0` here because using host, and setting it to `0.0.0.0` breaks the self test. Why not always bind to 0.0.0.0?